pkg/framework/version_bundle.go

```diff
@@ -147,12 +147,22 @@ func extractAuthorities(content, vType string) ([]versionbundle.Authority, error
 		return nil, microerror.Mask(err)
 	}
 
+	if len(indexReleases) < 2 {
+		return nil, microerror.Maskf(notFoundError, "release index length must be bigger than 2 but got %d", len(indexReleases))
+	}
+
+	// TODO At some point we should get rid of "wip" and "current" wording in the code below.
+	//
+	//	See https://github.com/giantswarm/giantswarm/issues/3313
+	//
 	sortedReleases := versionbundle.SortIndexReleasesByVersion(indexReleases)
 	sort.Sort(sort.Reverse(sortedReleases))
-	for _, ir := range sortedReleases {
-		if vType == "wip" && !ir.Active || vType == "current" && ir.Active {
-			return ir.Authorities, nil
-		}
+	switch vType {
+	case "latest", "wip":
+		return sortedReleases[0].Authorities, nil
+	case "previous", "current":
```

Comment:
I thought about replacing `previous` with `stable`. It makes more sense when we have versioning of operators in place. Also it has the same length as `latest` which will look nice in AWS console :D 

I'm not big on that though. WDYT?

Reply:
`stable` suggests some version semantic like `wip` and `cur`, which are both not correct. I feel the same alongside with `stable`. This is actually about numeric order. We are interested in the transitioning from one version to another. Regardless their semantic classification. We want to ensure that we are able to move on. This is about order. `latest` and `previous` makes sense here.

Reply:
Now yes. But probably not when we have versioning of operators in place. But then we probably have to change this code anyway.